### PR TITLE
feat: upgrade schema to 1.1.1 for sampleTag

### PR DESCRIPTION
**Pull request by chagong**
feat: upgrade schema to 1.1.1 for sampleTag

https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/24599180

### CODE REVIEW

**Review by hund030 — APPROVED** [review]

**Review by KennethBWSong — DISMISSED** [review]

**Review by SLdragon — DISMISSED** [review]

File: NPM-search-connector-M365/teamsapp.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
-# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.0.0/yaml.schema.json
-version: 1.0.0
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.1.1/yaml.schema.json
+version: 1.1.1
+sampleTag: TeamsFx-Samples:npm-search-connector
```
Comment (swatDong) [inline NPM-search-connector-M365/teamsapp.yml:3]: -M365

File: stocks-update-notification-bot/teamsapp.local.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
-# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.0.0/yaml.schema.json
-version: 1.0.0
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.1.1/yaml.schema.json
+version: 1.1.1
+sampleTag: TeamsFx-Samples:stocks-update-notification-bot
```
Comment (swatDong) [inline stocks-update-notification-bot/teamsapp.local.yml:3]: BTW, any plan for the dotnet sample?

**Review by wenytang-ms — DISMISSED** [review]

File: graph-connector-app/teamsapp.local.yml [hunk 3]
```diff
@@ -1,5 +1,6 @@
-# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.0.0/yaml.schema.json
-version: 1.0.0
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.1.1/yaml.schema.json
+version: 1.1.1
+sampleTag: TeamsFx-Samples:graph-connector-app
```
Comment (formulahendry) [inline graph-connector-app/teamsapp.local.yml:3]: Though this change for teamsapp.local,yml  has no side effect, but I think only changing teamsapp.yml is enough.

**Review by swatDong — APPROVED** [review]

**Review by KennethBWSong — APPROVED** [review]

File: stocks-update-notification-bot/teamsapp.local.yml (hunk 2 above)
Comment (chagong) [reply stocks-update-notification-bot/teamsapp.local.yml:3]: same change will happen

**Review by formulahendry — APPROVED** [review]

**Review by hund030 — APPROVED** [review]

**Review by SLdragon — APPROVED** [review]

**Review by huimiu — APPROVED** [review]

**Review by JerryYangKai — APPROVED** [review]

**Review by wenytang-ms — APPROVED** [review]

**Review by yukun-dong — APPROVED** [review]

File: graph-connector-app/teamsapp.local.yml (hunk 3 above)
Comment (chagong) [reply graph-connector-app/teamsapp.local.yml:3]: If the main "teamsapp.yml" gets loaded for every command, it should be ok for just adding to teamsapp.yml. But anyway, I just want to make sure all actons to local and main yml will have this property.

**Review by XiaofuHuang — APPROVED** [review]

**Comment by dooriya** [comment]
@chagong How about the codespaces samples:
- hello-world-tab-codespaces
- NPM-search-message-extension-codespaces
- notification-codespaces

**Comment by chagong** [comment]
> 



> 

> 



> 



> @chagong How about the codespaces samples:
> 
> * hello-world-tab-codespaces
> * NPM-search-message-extension-codespaces
> * notification-codespaces

will have another PR to update.

**Pull request merged**